docs/application/native/guides/connectivity/nfc.md

```diff
@@ -98,6 +98,13 @@ The main features of the NFC API include:
 
   ![HCE routing](./media/nfc_hce_routing.png)
 
+  > **Note**
+  >
+  > Pay attention to the following:
+  > - HCE operation requires launching HCE application in Tizen Application Framework. Because of this, enough time is required for the data to be transferred to the application.
```

Comment:
HCE operations begin with the launch of HCE application in Tizen Application Framework. With the execution of these operations, the data transfer to the application needs more time.